examples/packages/benchmarks/src/ssr/playground.tsx

```diff
@@ -2,15 +2,17 @@ import React, { useContext, createContext } from 'react';
 
 const Cache = createContext<Record<string, true> | null>(null);
 
-export const useCache = () => {
+export const useCache = (): Record<string, true> => {
```

Comment:
this return type seems sus, also wouldn't it better be `typeof Cache` ?

Reply:
potato potato

its smelly but its the same type after all. also this is in benchmarks who cares hehe